src/Oro/Bundle/UserBundle/Controller/UserController.php

```diff
@@ -130,14 +131,18 @@ public function deleteAction($id)
 
 
     /**
-     * @param User   $entity
+     * @param mixed  $user
      * @param string $updateRoute
      * @param array  $viewRoute
+     *
      * @return array
      */
-    protected function update(User $entity, $updateRoute = '', $viewRoute = array())
+    protected function update($user, $updateRoute = '', $viewRoute = [])
     {
-        if ($this->get('oro_user.form.handler.user')->process($entity)) {
+        if (!$user instanceof UserInterface) {
```

Comment:
in which cas $user can be an interface of UserInterface ?
If you remove force type on $entity, sf will never convert your id to a real entity